src/main/resources/assets/computercraft/recipes/skull_cloudy.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "minecraft:crafting_shapeless",
     "ingredients": [
-        { "item": "minecraft:skull", "data": 1 },
+        { "item": "minecraft:skull", "data": 32767 },
```

Comment:
The wildcard metadata is deprecated by Mojang, it will no longer work in 1.13 and does not show properly in the recipe book in 1.12. Mojang replaced all uses of it with a list of ingredients. Since its reused, male that list of skulls a constant

Reply:
Ahhh, thanks. I've used it without problem with JEI, but forgot to test the recipe book.

Out of interest, what's the recommended way of dealing with items like wool, where there's no corresponding ore-dict entry? Obviously in 1.13 we can use item tags, but it also seems a little sub-optimal to list all 16 variants.

Reply:
Yeah, all recipes that use all 16 variants of wool in 1.12 listed all 16 variants. You could also just add your own oredict entry to the vanilla items to supplement until tags are added in 1.13, as I'm sure other mods add their own wools too for the oredict